File: algorithm/src/main/java/baekjoon/p09498/Main.java
package baekjoon.p09498;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int input = sc.nextInt();
        System.out.println(solve(input));
    }

    private static String solve(int input) {
        switch (input / 10) {
            case 10:
            case 9:
                return "A";
            case 8:
                return "B";
            case 7:
                return "C";
            case 6:
                return "D";
            default:
                return "F";
        }
    }

}

File: algorithm/src/main/java/baekjoon/p09498/Main.kt
package baekjoon.p09498

fun main(args: Array<String>) {
    val input = readLine()!!.toInt()
    println(solve(input))
}

private fun solve(input: Int): String {
    when (input) {
        in 90..100 -> return "A"
        in 80..89 -> return "B"
        in 70..79 -> return "C"
        in 60..69 -> return "D"
    }

    return "F"
}